web/cobrands/peterborough/base.scss

```diff
@@ -37,6 +37,91 @@ h1, h2 {
     color: $alt-green;
 }
 
+.btn-primary,
```

Comment:
Are these changes going to affect existing Peterborough / WasteWorks pages, and if so, do we want that?

Reply:
I think we should do it, for the following reasons:

- I added a fix for the hover and active state of the primary button. Basically, the text wasn't accessible with the lighter background. So now we have the same effect the PCC council has on their website. Same with the focus effect.
- The bulky waste form is a bit more complex and involves more steps if we compare it with FMS and the current journeys on WW. These new elements will impact the UX journey, like highlighting what we want to highlight and giving more priority to the sections we want to emphasise. Are the 100% essential? No, but they are an improvement.
- I tested this PR on FMS and didn't see style conflict.